theano/sandbox/cuda/opt.py

```diff
@@ -741,11 +749,11 @@ def local_gpu_gemm(node):
         y_on_gpu = (y.owner and isinstance(y.owner.op, HostFromGpu))
         z_on_gpu = (z.owner and isinstance(z.owner.op, HostFromGpu))
         if x_on_gpu or y_on_gpu or z_on_gpu:
-            return [host_from_gpu(gpu_gemm_no_inplace(as_cuda_ndarray_variable(z),
-                                                 a,
-                                                 as_cuda_ndarray_variable(x),
-                                                 as_cuda_ndarray_variable(y),
-                                                 b))]
+            return [host_from_gpu(gpu_gemm_no_inplace(gpu_from_host(z),
+                                                      a,
+                                                      gpu_from_host(x),
+                                                      gpu_from_host(y),
```

Comment:
Oh my! That was only intended to be pep8 fixes. It's probably due to a bad merge I made where I didn't realize new master version was using as_cuda_ndarray_variable and not gpu_from_host. I'll fix this!